os/x11/window.h

```diff
@@ -128,8 +128,7 @@ class WindowX11 : public Window {
   bool m_resizable = false;
   bool m_transparent = false;
 
-  KeySym m_pressedKeySym = 0;
-
+  std::map<KeySym, bool> map_keys;
```

Comment:
I think it would be better if this is a globa/static/defined in the anonymous namespace of window.cpp, and it's called `g_pressedKeys`. And it might be an `std::set`.

Reply:
Done, also please check out https://github.com/aseprite/aseprite/pull/4080 - this fixes a behaviour when we for example press Shift + Key that got mapped to mouse trigger button, and then we release the key that got mapped but never release Shift. I've explained it in the PR's comments.